From 78ceb1bb808fb28cb88792f1166e655159630f8e Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Wed, 12 Feb 2020 13:14:28 -0600
Subject: [PATCH 034/120] arm64: dts: agilex: correct FPGA manager driver's
 compatible value

commit b23760380a9d29af0c927c1d780a6d8181566a0b from
https://github.com/altera-opensource/linux-socfpga.git

[upstream commit 5982561221780eb6872ea310cf0517e6ace27e61]

Correct the compatible property value for FPGA manager driver on
Intel Agilex SoC platform.

Signed-off-by: Richard Gong <richard.gong@intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 arch/arm64/boot/dts/intel/socfpga_agilex.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
index 3a8da29770a4..97701aad0cf8 100644
--- a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
+++ b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
@@ -624,7 +624,7 @@
 				memory-region = <&service_reserved>;
 
 				fpga_mgr: fpga-mgr {
-					compatible = "intel,stratix10-soc-fpga-mgr";
+					compatible = "intel,agilex-soc-fpga-mgr";
 				};
 			};
 		};
-- 
2.31.1

